ssntp/ssntp.go

```diff
@@ -1151,14 +1152,41 @@ func (config *Config) port() uint32 {
 	return port
 }
 
+func discoverCertificate(certPrefix string) (string, error) {
```

Comment:
So here we should avoid relying on specific certificate prefixes and names. Right now they're consistent because they're all generated by ciao-cert, but in the future we can't make that a hard requirement.
So what I'd prefer to see is a `getDefaultCertificate() (string, string, error)` function that would return both certificate paths when found. It would essentially do what `discoverCertificate()` does plus the following:
1. Actually open and read each of the 2 PEM files found
2. Verify that 1 is a CA and the other is not.
If that is the case, we return the 2 paths found.
Eventually we will also do:
3. Verify the non CA cert with the CA one

After building an io reader from a certificate file and getting a byte slice from it (`certPEM`), you have to do the following for implementing step 2:
```
	block, _ := pem.Decode(certPEM)
	if block == nil {
		panic("failed to parse certificate PEM")
	}
	cert, err := x509.ParseCertificate(block.Bytes)
	if err != nil {
		panic("failed to parse certificate: " + err.Error())
	}
```

Then you have a `x509.Certificate` structure that represents a decoded PEM block. Testing if it's a CA or not is verifying: `cert.IsCA == true`.